packages/bpk-stylesheets/webpack.config.babel.js

```diff
@@ -21,7 +21,6 @@ import WrapperPlugin from 'wrapper-webpack-plugin';
 
 import postCssPlugins from '../../scripts/postCssPlugins';
 
-import babelConfig from './base.babel.config';
```

Comment:
We were only using this for support for IE11 which we no longer support in Skyscanner